apps/library/lib/library/game.ex

```diff
@@ -57,28 +59,24 @@ defmodule Library.Game do
     # do:  {:reply, {:ok, word}, board_state}
     # else {:reply, {:error, "word not valid", board_state}}
 
-    if true do
-      new_board = Enum.reduce(word, board_state.board, fn(letter, acc) ->
-        replace_letter(letter, player, acc) 
-      end)
-
-      new_state = %{board_state | board: new_board}
-
-      {:reply, :ok, new_state}
-    else
-      {:reply, {:error, "reason"}, board_state}
-    end
-  end
+    new_board = Board.add_word(atomize_word(word), player, board_state.board)
+    new_state = %{board_state | board: new_board}
```

Comment:
maybe we should change `board_state` to `game_state` since we've now separated their concerns a bit more